packages/block-editor/src/components/link-control/settings-drawer.js

```diff
@@ -25,8 +25,7 @@ function LinkSettingsDrawer( { children, settingsOpen, setSettingsOpen } ) {
 	return (
 		<>
 			<Button
-				// TODO: Switch to `true` (40px size) if possible
-				__next40pxDefaultSize={ false }
+				__next40pxDefaultSize
```

Comment:
**Before:**
![link-control-4-before](https://github.com/user-attachments/assets/39fa46bd-d931-49f4-a193-ef77b89737df)

**After:**
![link-control-4-after](https://github.com/user-attachments/assets/0be32f30-7727-4ed7-a667-850d1a810231)

Reply:
I think this one looks good 👍